src/equisolve/numpy/preprocessing/_base.py

```diff
@@ -177,26 +168,19 @@ def fit(
                     properties=Labels.single(),
                 )
 
-            for parameter in self.parameter_keys:
-                if parameter == "values":
-                    continue
+            for parameter in X_block.gradients_list():
```

Comment:
this change was required because we allow to only apply the standardizer on certain gradient keys using parameter_keys, so if parameter_keys does not contain all keys, later the returned TensorBlock only contained the gradients in paramater_keys.  That was not the intention, so we just the means to zero and scale to 1 when parameter is not in parameter_keys

Reply:
This makes me think the usage of `parameter_keys` is a bit fragile. I guess this can be addressed when re-writing this class with operations, so it is fine for now.

Could you remind me why this functionality is not built on top of `remove_gradients`? Is there something missing?

Reply:
We thought that a user might for example want to only fit on energies, but wants to also predict forces (for a quick and dirty model) without removing the gradients, because we still want to output the force error at the end. I think it can be discussed how useful this actually is, but I wouldn't touch it in this PR.